### ISSUE TITLE
Lesson pilot at the Netherlands eScience Center 11, 12, 13 december 2023

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/python-intermediate-development#259]
We will have another lesson pilot at the Netherlands eScience Center, in-person at 11, 12 & 13 december 2023.

We will do the 5 sections, each in half a day. And end with a 6th half day of 'apply what you learned on your own project'.

Please note we already piloted before #159

**Comment by anenadic** [comment]
Thank you @svenvanderburg - please report back any feedback and the number of learners when you get a chance after the workshop. Good luck! 🍀

**Comment by svenvanderburg** [comment]
@anenadic the workshop went really well! Some improvements we did that worked well since the last run:
- A 3-day workshop instead of squeezing everything in 2 days
- A 6th half-day section 'apply what you learned on your own project' which worked out really well
- To accommodate slow learners we made some episodes optional (such as the functional and OOP episodes) and clearly communicate this.
- To accommodate fast learners we had optional extra in-depth exercises for each section
- We started each section with a 30 minute introduction to all topics in the upcoming section. We find this helps participants to get through the material better and reduce the amount of reading (they can skim through some of the material because we already introduced it). 
- We have frequent temperature checks to see how far everyone is in the material and make it clear when we start with the next section. We start a new section when everyone (almost) finished the previous section. We often let participants vote if they want to spend more time in a section or go to the new one. This works really well!

The main feedback:
- We consistently get feedback that it is too much reading, stressing the importance to pick up: #260 . 
- Some small outdated parts of the lesson material, we will submit a PR shortly.
